src/components/forms/DatePicker/Calendar.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+
+import { Calendar } from './Calendar'
+import { FocusMode } from './DatePicker'
+import { parseDateString } from './utils'
+
+export default {
+  title: 'Components/Form controls/Date picker/Calendar',
+  component: Calendar,
+  argTypes: {
+    handleSelectDate: { action: 'select date' },
+    setStatuses: { action: 'set statuses' },
```

Comment:
Right now all the` <Calendar> `and `<YearPicker>` stories break for me in Storybook when I try to interact with the components `TypeError: setStatuses is not a function`. Perhaps Storybook actions don't work nicely for mocking behavior in this case? 

Happy to pair on this and figure out a workaround. Or, another idea, we could comment out the export line for stories that aren't reflected in components we export from `index.ts`. Seems like these stories are more for our internal use as library developers, we could comment them in when we need to check stuff, but otherwise I don't think they need to go into the published Storybook.